.github/workflows/cypress_tests.yml

```diff
@@ -96,6 +75,21 @@ jobs:
           role-duration-seconds: 3600
           role-session-name: OPGLPACypressTestsCypressS3Monitor
 
+      - uses: actions/cache@v2
+        with:
+          path: |
+            node_modules
+          key: node-cache-${{ runner.os }}-${{ hashFiles('package-lock.json') }}
+
+      - uses: actions/setup-node@8c91899e586c5b171469028077307d293428b516 # pin@v3.5.1
+        with:
+          node-version: 16
+
+      - name: Cypress install
+        uses: cypress-io/github-action@v5.0.2
```

Comment:
Does this fix us to a specific version of cypress, or is this just so we get all the necessary dependencies installed?

Reply:
I've removed this now so we're just using the version of Cypress in the package.json file